plugins/wiki/app/views/pages/all.html.haml

```diff
@@ -6,7 +6,6 @@
 .navbar
   .navbar-inner
     %ul.nav
-      %li= link_to t('.recent_changes'), all_pages_path(:view => 'recent_changes')
```

Comment:
Ah, now I get it. I guess it's better like this - hope people will not get confused.